app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +29,10 @@
       android:name=".mydownloads.MyDownloadsActivity"
       android:screenOrientation="portrait"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".onBoarding.OnBoardingActivity"
```

Comment:
Update the package name to `onboarding` considering that we are following this convention with other package names too.

Reply:
Done.